src/test/scala/io/iohk/ethereum/network/p2p/messages/TransactionSpec.scala

```diff
@@ -28,12 +28,22 @@ class TransactionSpec extends FlatSpec with Matchers {
     signatureRandom = ByteString(Hex.decode("cfe3ad31d6612f8d787c45f115cc5b43fb22bcc210b62ae71dc7cbf0a6bea8df")),
     signature = ByteString(Hex.decode("57db8998114fae3c337e99dbd8573d4085691880f4576c6c1f6c5bbfe67d6cf0")))
 
+  val newInvalidStx = SignedTransaction(
```

Comment:
maybe test name is wrong, it should file when we trying to get global transaction and just by changing point sign encoding change it to chain specific transaction
chain specific transaction have to include chain id as part of signed message